tests/simplex/step-06.cc

```diff
@@ -0,0 +1,397 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Step-06 on a simplex mesh.
+//
+// The following changes had to be made due to incompatibilities:
+//  - Manifolds are disabled
+//    All manifold ids from the triangulation object generated with
+//    GridGenerator::hyper_ball() had to be removed
```

Comment:
Why? Could you do it like this: https://github.com/dealii/dealii/blob/b59c748f39b00cf111ab9eec96c115d8aa9792a2/tests/simplex/step-01.cc#L65-L67

Reply:
Will give it a try!